src/state/store.js

```diff
@@ -16,18 +16,26 @@ const makeSubject = () => {
 
 const defaultOptions = {
   throwOnValidation: false,
-  throwOnMissingSchemas: false
+  throwOnMissingSchemas: false,
+  batchUpdateFn: requestAnimationFrame,
```

Comment:
What if we just default the function to `fn => fn()`? That way, there's only one path through the code, regardless of whether the function is synchronous or not.